Gomfile

```diff
@@ -3,7 +3,8 @@ gom 'github.com/gorilla/mux', :commit => 'e444e69cbd2e2e3e0749a2f3c717cec491552b
 gom 'github.com/gorilla/context', :commit => '215affda49addc4c8ef7e2534915df2c8c35c6cd'
 gom 'gopkg.in/unrolled/render.v1', :commit => '40ac5716c7e0bffcc440aac649ea4b880a5d7735'
 gom 'github.com/alext/tablecloth', :commit => '7f49e40'
-gom 'github.com/alphagov/plek/go', :commit => 'f5da97e'
+gom 'github.com/alphagov/plek/go', :commit => 'c0c56fb'
```

Comment:
A note.  gom does support pinning to tags (with `:tag => 'v1.8.1'`) - see https://github.com/mattn/gom#gomfile.

Not a blocker for merge, just worth noting...